makefiles/libc/newlib.mk

```diff
@@ -19,46 +19,57 @@ endif
 
 export LINKFLAGS += -lc
 
+# Note on `realpath` vs `abspath`
+#
+# `realpath` is used instead of `abspath` to support Mingw32
+# It fixes issues when using windows formatted paths like in the gcc output
+# It does leave the trailing `/` on windows though
+
+
 # Search for Newlib include directories
 
 # Try to search for newlib in the standard search path of the compiler for includes
 ifeq (,$(NEWLIB_INCLUDE_DIR))
-  COMPILER_INCLUDE_PATHS := $(shell $(PREFIX)gcc -v -x c -E /dev/null 2>&1 | grep '^\s' | tr -d '\n')
```

Comment:
Why you assume GCC is installed?

Reply:
It was already the case before.

Also it is not possible to compile for `llvm` without `gcc`.

https://github.com/RIOT-OS/RIOT/blob/master/makefiles/toolchain/llvm.inc.mk
https://github.com/RIOT-OS/RIOT/blob/master/makefiles/libc/newlib.mk#L65